packages/react-inbox/src/components/Message/index.tsx

```diff
@@ -58,44 +60,44 @@ const Message: React.FunctionComponent<MessageProps> = ({
     });
   }, [created]);
 
-  const actions = useMemo(() => {
-    return [{ href: data?.clickAction, label: "View Details" }];
-  }, [data]);
+  const actions = useMemo(() => [{
+    href: data?.clickAction,
+    label: "View Details",
+  }], [data]);
 
   const options = useMemo(
     () =>
       [
         !read &&
           trackingIds?.readTrackingId && {
-            label: "Mark as Read",
-            onClick: () => {
-              if (!trackingIds?.readTrackingId) {
-                return;
-              }
+          label: "Mark as Read",
```

Comment:
this spacing looks off

Reply:
<img width="595" alt="Screen Shot 2021-03-30 at 2 24 59 PM" src="https://user-images.githubusercontent.com/16184018/113058987-bd616a80-9163-11eb-8f43-3eb7ca7c29b7.png">

My editor gives me an error. Wonder if we have different settings

Reply:
we should get consistent here, newline with an object should be indented